uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -410,7 +411,14 @@ Arguments:
 sub PatientNameMatch {
     my $this         = shift;
     my ($dicom_file) = @_;
-    my $cmd          = "dcmdump $dicom_file | grep PatientName";
+
+    my $lookupCenterNameUsing = NeuroDB::DBI::getConfigSetting(
+        $this->{'dbhr'},'lookupCenterNameUsing'
+    );
```

Comment:
@cmadjar @nicolasbrossard 

now that we are getting it from the database, shouldn't we check if this setting is defined before running dcmdump?

Reply:
I think we should. The new DB framework will check that by the way (it will throw an exception if the setting was not found).

Reply:
if I put the word 'test' in that Config setting (which clearly does not exist in DICOMs), running the script would give me this error message:

```
W: unrecognized tag name: 'test'
Spool message is: 
The patient name cannot be extracted 
```

Reply:
also in the Loris front end, this setting is called Lok UpCenterName using because the getPSC() uses it, and this is what the hover over says. Shouldn't it somehow also reflect that we are now going to use it to validate DICOMs based on the proper Loris identifier?

Reply:
oooooh, I did not think of testing a random string for the Config setting. Thanks!! 👍 

So, if I add the -q option to the dcmdump command, then you don't see the error from dcmdump but you can get into the if statement that checks if a value was returned. So now, if I put 'test' in the lookupCenterNameUsing config setting, it will print a nice error message:
```
The 'test' DICOM field cannot be extracted from the DICOM file /data/not_backed_up/ImagingUpload-11-47-w6aIf0/MTL0609_138233_PREBL00_20170410_154459088_adniT1/MR.1.3.12.2.1107.5.2.32.35442.2017041015351355915460270
```
But if I have a correct DICOM field, then it runs properly.

Reply:
@MounaSafiHarab I just sent a PR on the LORIS side to change the description of the Config setting. Here is the PR: https://github.com/aces/Loris/pull/3585

Thanks!